src/enlyze/validators.py

```diff
@@ -24,6 +28,24 @@ def _ensure_datetime_aware(dt: datetime) -> datetime:
     return dt.astimezone(timezone.utc)
 
 
+def validate_datetime(dt: datetime) -> datetime:
+    if dt.utcoffset() is None:
+        logging.warning(_NAIVE_DATETIME_DISCOURAGED_LOG_MESSAGE)
```

Comment:
This line has no test coverage.

Reply:
Yes, but what would I be testing?

Reply:
Doesn't matter, but we want to keep test coverage at 100%, otherwise it becomes a slippery slope. In cases where tests make no sense or something cannot be tested but is reported as uncovered, there's always the option to exclude it from the coverage report. But here it's easy, have a test case that validates a naive timestamp.

Reply:
For the sake of 100% test coverage, 4ab438b defines a naive datetime object as an example :grimacing: